File: solutions/java/CreatePhoneNumber.java
package answers;

import java.util.stream.IntStream;

public class CreatePhoneNumber {
	public static void main (String[] args) {
		System.out.println(solution(new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 0}));
	}
	
	private static String solution(int[] s) {
		return String.format("(%d%d%d) %d%d%d-%d%d%d%d", IntStream.of(s).boxed().toArray());
	}
}

File: solutions/haskell/CreatePhoneNumber.hs
createPhoneNumber :: [Int] -> String
createPhoneNumber l = "(" ++ get 0 ++ get 1 ++ get 2 ++ ") " ++ get 3 ++ get 4 ++ get 5 ++ "-" ++ get 6 ++ get 7 ++ get 8 ++ get 9
    where get n = show (l !! n)
